README.md

```diff
@@ -1603,28 +1602,53 @@ We reported this issue in the Github Helm repository [[5](#5)], but we have not
 
 #### 4.5.1.1. Docker image preparation
 
-`build.sh` script can be used to compile all the services and generate the Docker images.
-When executed, the following steps are performed:
-* Build `proto-idls` project
-* Build `restapi` project & generate Docker image
-* Build `stockchecker` project & generate Docker image
-* Build `dal` project & generate Docker image
+Our project provides a bash script called `build.sh` to compile all the services and generate their Docker images. When executed, the following steps are performed:
+
+- Build `proto-idls` project and install generated JAR file in maven repository (this project keeps common classes shared by the rest of the projects)
+- Build `restapi` project & generate Docker image
+- Build `stockchecker` project & generate Docker image
+- Build `dal` project & generate Docker image
+
+#### 4.5.1.2. Kubernetes deployment
+
+Once all the Docker images of the services have been generated we may proceed to start the cluster in Kubernetes. As Prometheus Operator chart leaves some spillovers and they have to be removed manually, we consider it would be useful to create a script (called `k8s.sh`) that will do some things automatically.
```

Comment:
"start the cluster in Kubernetes" 
maybe something like:
"deploy the application in Kubernetes cluster"